docs/home/whats-new-2-15.md

```diff
@@ -1,94 +1,103 @@
+      * Trainings form
+      * Environments form
+  
+    You can select `single` or `multi-node (distributed)` training. When configuring distributed training, you will need to select a framework from the list. For *Trainings* configuration, see [Adding trainings](../Researcher/user-interface/trainings.md#adding-trainings). See your Run:ai representative to enable this feature. For *Environments* configuration, see [Creating an Environment](../Researcher/user-interface/workspaces/create/create-env.md#creating-a-new-environment).
 
-<!-- RUN-11206/RUN-11242 Submitting distributed training from UI - 2nd form page-->
+<!-- ADDLINK RUN-10411/RUN-11390 Support self-signed certificates-->
+* Run:ai can be installed in an isolated network. In this air-gapped configuration, the organization will not be using an established root certificate authority{target=_blank}. Instead, the organization creates a local certificate which serves as the root certificate for the organization. The certificate is installed in all browsers within the organization. For more information, see [link here](##org-cert.md).
 
-<!-- RUN-11219/RUN-11602 Submitting an MPI distributed training from UI - 3rd form page-->
+<!-- RUN-10451/RUN-10452 Support new Kubernetes and OpenShift releases - Q3/2023-->
+* Updated the compatibility matrix to include supported versions for Kubernetes and OpenShift. For more information, see [Cluster prerequisites](../admin/runai-setup/cluster-setup/cluster-prerequisites.md#kubernetes)
 
-<!-- RUN-11231/RUN-11240 Environment for distributed training-->
+--8<-- "home/whats-new-2-14.md:41:43"
 
 <!-- RUN-11282/RUN-11283 Nodepools enabled by default-->
+* Improvement in node pools which are now enabled by default. There is no need to enable the feature in the settings.
 
 <!-- RUN-11292/RUN-11592 General changes in favor of any asset based workload \(WS, training, DT\)-->
-
-<!-- RUN-11325 R&D Support time Q3-->
-
+* Improved the *Trainings* and *Workspaces* forms. Now the runtime field for *Commands* and *Arguments* can be edited even after it has inherited it from the environment.
 
 <!-- RUN-11525/RUN-11538 Support Kubernetes non-privileged PSA on project namespaces for Openshift-->
+* Added support for non-privileged PSA on project namespaces for OpenShift environments.
 
-<!-- RUN-11610 Sunset of old submit form-->
+<!-- ADDLINK RUN-11692/RUN-11694 Scoping for template-->
+* Added support for *Scope* in the template form. For configuration information, see [Creating templates]().
 
-<!-- RUN-11692/RUN-11694 Scoping for template-->
-
-<!-- RUN-11974 Use Secure images only \(UBI\)-->
-
-<!-- RUN-12166 Cost fixes-->
-
-<!-- Include from 2.14 -->
-
-<!-- TODO Adjust the whats new for 2.14 to add in the sections for New features and Improvements. Then include those in the appropriate sections for 2.15 after the primary ones are written up.
---8<-- "home/whats-new-2-14.md:4:69"
--->
```

Comment:
@jasonnovichRunAI aren't we mentioning newly supported K8S and Openshift versions?

Reply:
@gal-revach If that is something you want to do. Usually, we do not mention it unless there is a significant need, or there is a EOL/EOS (end of life/end of support) for particular versions.

Reply:
We do have deprecated versions. If we mention them than maybe we can mention the new supported versions as well:
Added support for Kubernetes 1.28 and Openshift 4.13
Removed support for Kubernetes 1.23, 1.24 and OpenShift 4.10